run.go

```diff
@@ -31,42 +29,65 @@ import (
 
 type runOptions struct {
 	stepName        string
-	signer          cryptoutil.Signer
+	signers         []cryptoutil.Signer
 	attestors       []attestation.Attestor
 	attestationOpts []attestation.AttestationContextOption
 	timestampers    []timestamp.Timestamper
+	insecure        bool
 }
 
 type RunOption func(ro *runOptions)
 
+// RunWithInsecure will allow attestations to be generated unsigned. If insecure is true, RunResult will not
```

Comment:
I think that while I am cautious that we need to introduce `insecure` with a high level of caution (really discourage users from doing it), `--insecure` is fine in my eyes. Just wondering what we can do to encourage signing over insecure further if we're going to introduce it. Maybe an extra log warning to specify that they are running as such and it is not an encouraged pattern to introduce.

Reply:
This is a nit so no action is required 👍